src/test/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImplTest.java

```diff
@@ -0,0 +1,122 @@
+package edu.cornell.kfs.pmw.batch.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.kuali.kfs.sys.KFSConstants;
+import org.mockito.Mockito;
+
+import edu.cornell.kfs.pmw.batch.PaymentWorksConstants;
+import edu.cornell.kfs.pmw.batch.businessobject.PaymentWorksVendor;
+import edu.cornell.kfs.pmw.batch.service.PaymentWorksFormModeService;
+
+class PaymentWorksVendorToKfsVendorDetailConversionServiceImplTest {
+    
+    private static final String LEGACY_PO_COUNTRY = "legacy po country";
+    private PaymentWorksVendorToKfsVendorDetailConversionServiceImpl conversionService;
+    private PaymentWorksVendor pmwVendor;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        Configurator.setLevel(PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.class.getName(), Level.DEBUG);
+        conversionService = new PaymentWorksVendorToKfsVendorDetailConversionServiceImpl();
+        pmwVendor = new PaymentWorksVendor();
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+        conversionService = null;
+        pmwVendor = null;
+    }
+    
+    private PaymentWorksFormModeService buildMockPaymentWorksFormModeService(boolean useLegacy, boolean useForeign) {
```

Comment:
are the useLegacy vs useForeign mutually exclusive? I wonder if you could just give as input a constant like USE_LEGACY or USE_FOREIGN. Then the other value would just be the opposite. Would make it a bit more clear in the tests what the setting is.

Reply:
for purposes of this test it is one or the other, I will clean up as suggested